docs/user-guide/monitoring.md

```diff
@@ -22,6 +22,8 @@ Running the following command deploys prometheus in Kubernetes:
 ```console
 kubectl apply --kustomize github.com/kubernetes/ingress-nginx/deploy/prometheus/
 ```
+!!! Helpful
+Detailed information about installation and configuration can be found [here](https://dev.to/xaviergeerinck/monitoring-the-kubernetes-nginx-ingress-controller-with-prometheus-and-grafana-35gi)
```

Comment:
@kundan2707 since this is not an official document, wouldn't it be a bad practice to point to third party documentation?

Reply:
@chirangaalwis should we create similar document which will be reviewed and then we can put in repo itself?

Reply:
@kundan2707 +1 for having a similar, official doc IMO.